common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -382,12 +384,13 @@ object CometConf {
       .createWithDefault(Seq("Range,InMemoryTableScan"))
 
   val COMET_ANSI_MODE_ENABLED: ConfigEntry[Boolean] = conf("spark.comet.ansi.enabled")
+    .internal()
```

Comment:
Why making it internal?

Reply:
As discussed in https://github.com/apache/datafusion-comet/pull/493#discussion_r1625073733
doc is not per profile yet. By making it internal, doc will ignore it.